docs/manual/kinds/ceos.md

```diff
@@ -83,6 +83,26 @@ When containerlab launches ceos node, it will set IPv4/6 addresses as assigned b
     ```
     As you see, the management interface `Ma0` inherits the IP address that docker assigned to ceos container management interface.
 
+
+## Additional Interface naming considerations
```

Comment:
do you mind to lowercase the Interface word?

Reply:
sure thing.  done.